### PR TITLE
More console fixes

### PR DESCRIPTION
**Pull request by andytaylor**

### CODE REVIEW

**Comment by clebertsuconic** [comment]
nit-pick...

NO-JIRA Updating the console docs

And the commit message.. if you could say the substantial part in the first line? It makes it a lot easier when we are looking on the logs...


 :)

**Comment by andytaylor** [comment]
@clebertsuconic nit pick fixed :)

File: artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/ActiveMQServerControlImpl.java [hunk 1]
```diff
@@ -3430,21 +3430,21 @@ public void addAddressSettings(final String address,
       addressSettings.setAutoDeleteQueues(autoDeleteQueues);
       addressSettings.setAutoCreateAddresses(autoCreateAddresses);
       addressSettings.setAutoDeleteAddresses(autoDeleteAddresses);
-      addressSettings.setConfigDeleteQueues(configDeleteQueues == null ? AddressSettings.DEFAULT_CONFIG_DELETE_QUEUES : DeletionPolicy.valueOf(configDeleteQueues.toUpperCase()));
-      addressSettings.setConfigDeleteAddresses(configDeleteAddresses == null ? AddressSettings.DEFAULT_CONFIG_DELETE_ADDRESSES : DeletionPolicy.valueOf(configDeleteAddresses.toUpperCase()));
+      addressSettings.setConfigDeleteQueues(configDeleteQueues == null || configDeleteQueues.length() == 0 ? AddressSettings.DEFAULT_CONFIG_DELETE_QUEUES : DeletionPolicy.valueOf(configDeleteQueues.toUpperCase()));
```
Comment (mpoutlook) [inline artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/ActiveMQServerControlImpl.java:3433]: why not use isEmpty()

File: artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/ActiveMQServerControlImpl.java [hunk 2]
```diff
@@ -3430,21 +3430,21 @@ public void addAddressSettings(final String address,
...
+      addressSettings.setConfigDeleteAddresses(configDeleteAddresses == null || configDeleteAddresses.length() == 0 ? AddressSettings.DEFAULT_CONFIG_DELETE_ADDRESSES : DeletionPolicy.valueOf(configDeleteAddresses.toUpperCase()));
       addressSettings.setMaxSizeBytesRejectThreshold(maxSizeBytesRejectThreshold);
-      addressSettings.setDefaultLastValueKey(defaultLastValueKey == null ? ActiveMQDefaultConfiguration.getDefaultLastValueKey() : new SimpleString(defaultLastValueKey));
+      addressSettings.setDefaultLastValueKey(defaultLastValueKey == null || defaultLastValueKey.length() == 0 ? ActiveMQDefaultConfiguration.getDefaultLastValueKey() : new SimpleString(defaultLastValueKey));
```
Comment (michaelpearce-gain) [inline artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/ActiveMQServerControlImpl.java:3436]: why not use isEmpty()?

Comment (andytaylor) [reply artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/ActiveMQServerControlImpl.java]: yip, updated

**Pull request merged**